website/docs/r/system_config.html.markdown

```diff
@@ -0,0 +1,45 @@
+---
+layout: "mso"
+page_title: "MSO: mso_system_config"
+sidebar_current: "docs-mso-resource-system_config"
+description: |-
+  Manages MSO System Configuration.
+---
+
+# mso_schema_site_vrf_region #
+
+Manages MSO System Configuration.
+
+## Example Usage ##
+
+```hcl
+
+resource "mso_system_config" "system_config" {
+  alias = "test alias"
+  banner {
+    message = "test message"
+    state = "active"
+    type = "warning"
+  }
+  change_control = {
+    workflow = "enabled"
+    number_of_approvers = 2
+  }
+}
+
+```
+
+## Argument Reference ##
+
+* `alias` - (Optional) The system Alias.
+* `banner` - (Optional) A list of Banner configuration. 
+    * `state` - (Required) The state of the Banner. Allowed values are `active` or `inactive`.
+    * `type` - (Required) The type of the Banner. Allowed values are `critical`, `warning` or `informational`.
+    * `message` - (Required) The message of the Banner.
+* `change_control` - (Optional) A map of Change Control configuration. 
+    * `workflow` - (Required) Whether Change Control workflow is enabled. Allowed values are `enabled`, or `disabled`.
+    * `number_of_approvers` - (Optional) The number of approvers for the Change Control. MSO defaults to `1` when not provided.
+
+## Attribute Reference ##
+
+No attributes are exported.
```

Comment:
I see that create behaves like import in certain way, but should we add an example of how?

Reply:
What would you like to add then? Because import does not exist as functionality since it is not defined. 

Possibility would be to add a note in the description that import is not supported. Also perhaps then add a note regarding that delete removes from state only since system config cannot be created/deleted.

Reply:
Adding a note sounds good.